types/utils.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import "strings"
+
+const (
+	// MainnetChainID defines the Haqq EIP155 chain ID for main network
+	MainNetChainID = "haqq_11235"
```

Comment:
that's not ChainId, it's just some part of id. Any person using this will get confused by wrong naming and as a result will programm wrong logic

Reply:
Also there is no reason to just check for prefixes. There should be exact match.